package.json

```diff
@@ -34,7 +34,27 @@
     "sqlite3": "^5.0.0"
   },
   "devDependencies": {
-    "nodemon": "^2.0.1"
+    "nodemon": "^2.0.1",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
```

Comment:
Can you remove this package and make sure it's removed from package lock as well? JSX is only needed for React and we aren't currently using React.

Reply:
"Can you add a command in package.json under scripts that lets a developer run the same command as the hook to test it out" - Not sure I follow this part.

Reply:
I'm not sure how to debug the hook but we can start with having an npm script that can lint all files. In `package.json` we can add something like:

```json
"scripts": {
    ...
    "lint": "npx eslint ./JS/**/*.js ./public/scripts/**/*.js"
  },
```

So we can run the linting separate from commit hook. That might help debug the issue of why it doesn't seem to be working. When I added that, the linting wasn't working so that may be why the hook isn't working either.

Reply:
How about just "lint": "eslint ./;" ? This is what I see in our repo and we use eslint.

Reply:
Also @vegetabill - I'm running into more issues when I run "npm run lint"

Error: Failed to load parser 'babel-eslint' declared in '.eslintrc': Cannot find module 'babel-eslint'

But when I remove it, it says, 

ESLint couldn't find the plugin "eslint-plugin-jsx-a11y".

So after adding that using npm install eslint-plugin-jsx-a11y@latest --save-dev, I get 401 problems (390 errors, 11 warnings) when I run linting. I think its looking at rules under /Users/swarna.gopalan/Hackathon/ghc/techtonica/keyboard-shortcuts-practice/node_modules/eslint-config-airbnb.
<img width="242" alt="Screen Shot 2020-10-02 at 3 22 46 PM" src="https://user-images.githubusercontent.com/5195759/94966656-448a9a00-04c3-11eb-81c9-1dcab5efc494.png">